data/presets/building/waste_shed.json

```diff
@@ -0,0 +1,27 @@
+{
+    "icon": "maki-recycling",
+    "fields": [
+        "{building}",
+        "lockable",
+        "access"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "locationSet": {
+        "include": ["DE", "EE", "SE"]
+    },
+    "tags": {
+        "building": "shed",
+        "amenity": "recycling",
+    },
+    "addTags": {    
+        "access": "private",        
+        "lockable": "yes"
```

Comment:
Not sure whether I would assume that

Reply:
In the Estonian community we thought locked might be the default. I don't see any way to make that region specific.

Reply:
In theory there could be separate one for Estonia and other one more general, reusing labels. But that seems quite overengineered.

Reply:
Well, I guess if you're adding hundreds of "waste sheds" then probably not all of them are locked, so it'd be better to have `lockable` as a featured field than as a field with a default value.